src/Pages/Home/Home.jsx

```diff
@@ -10,18 +10,30 @@ import { useNote } from "../../Contexts/notesActions-context";
 import { useState } from "react";
 import { useAuth } from "../../Contexts/authentication-context";
 import { addNote } from "../../Utilities-Functions/addNote";
+import { editNote } from "../../Utilities-Functions/editNote";
 
```

Comment:
You can export all utilities from a single index file and import them from that file here. Reduces number of lines of code for imports